terraform/modules/enclave/README.md

```diff
@@ -32,19 +32,17 @@ Follow these steps to run infrastructure tests:
 
 ## Deploying
 
-When performing a deployment you need to ensure you have the correct permissions to do so as extra permission are required.
```

Comment:
According to @kentsanggds, this is relevant to the Verify enclave, not our EC2 enclave. It would be good to specify that, and what extra permissions are needed. I have no idea.

Reply:
Correct.

Permissions wise you need to have a aws-vault profile as such:

```
[profile verify-perf-a]
mfa_serial = arn:aws:iam::<user_arn>:mfa/<email_address>
role_arn = arn:aws:iam::170611269615:role/prometheus_deployer
```

I believe as well that team members will also need to be added to have access for that role. I don't know if this is something we can do or something we need to ask people with Verify permissions to do for us.

https://reliability-engineering.cloudapps.digital/iaas.html#access-aws-accounts